docs/testing.md

```diff
@@ -0,0 +1,123 @@
+
+### Verify the Command Results
+After you build the command, verify that you have received the results that you expect:
+
+1. Open the **Task Library** and select **Create Task**.
+2. Configure the new task.
+
+| Configuration | Action |
+| ---- | ----| 
+| **Conditional** | Select the **Conditional** button to display the condition options. |
+| **Task Name** | Type a task name. |
+| **From previous tasks** |  Click **{}** to display the **Select source for** tool. The **Select source for** tool displays the **#2 ip** task that you created. |
+| **2 ip** | Click to display the **ip** task configurations. |
+| **IP** | Click **Address** and click **Close**. The `IP.Address` is displayed in the **From previous tasks** field. This is the Context Path. |
+| **From previous tasks** | Wrap the Context Path using this format `${IP.Address}`. Wrapping the Context Path tells Demisto to retrieve the value located in the curly brackets. |
+| **As value** | Type 8.8.8.8 and click ✅. |
+
+**Note:** If you need to edit the value in a field, you can click on the value and edit it. For example, click on the value in the **From previous tasks** field and edit the `${IP.Address}` value.
+
+3. Optional: If you need to filter or format the result, click **Filters and Operations** located in the **Select source for** dialog box.
+4. Click **Save**.
+5. Connect the **ip** task to the **Verify Command Results** task.
+
+### Close the Investigation
+1. Navigate to **Playbooks** and click **New Playbook**.
+2. In the search field, type **closeinvestigation** and click **BuiltIn Commands**.
+3. For **closeInvestigation**, click **Add**.
+4. Click the **{}** in the *id* field.
+5. Click **Incident details** and find **ID**. `${incident.id}` is inserted into the **id** field.
+6. Click **Close** and click **OK**.
+7. Connect the **Verify Command Results** task to the **closeInvestigation** task.
+8. Choose the **yes** label name for the condition and click **Save**.
+
+### Naming and Exporting the Playbook
+Demisto uses a standard naming convention for playbook tests that follows this format: `Integration_Name-Test`.
+
+1. Click **Save Version**.
+2. Exit the playbook editor.
+3. Export the playbook by clicking ![download button](https://user-images.githubusercontent.com/42912128/50277516-4d74bd80-044d-11e9-94b6-5195dd0db796.png).
+
+## Adding the Playbook to your Project
+1. In the YAML file that you created, edit the `id` so that it is identical to the `name` field.
+2. Modify the value in the `version` field to *-1* to prevent user changes.
+3. Using the example above, the top of your YAML should look like this:
+
+```yml
+id: IPInfo-Test
+version: -1
+name: IPInfo-Test
+```
+
+## Adding Tests to conf.json
+In order to associate integrations with a test plabyook we mange a `conf.json` file (at the root of the repository). The conf.json file is located in the **Tests** directory.
+
+The following is an example of a correct conf.json entry for an integration:
+```yml
```

Comment:
Suggest using `json` instead of `yaml` here to fix rendering.

Reply:
It's ok. It needs to be yml. This is what we use for playbooks.